packages/@react-types/checkbox/src/index.d.ts

```diff
@@ -72,7 +74,7 @@ export interface CheckboxProps extends ToggleProps {
 
 export interface AriaCheckboxProps extends CheckboxProps, AriaToggleProps {}
 
-export interface AriaCheckboxGroupProps extends CheckboxGroupProps, DOMProps, AriaLabelingProps, AriaValidationProps {
+export interface AriaCheckboxGroupProps extends CheckboxGroupProps, DOMProps, AriaLabelingProps, AriaValidationProps, HelpTextProps, Validation {
```

Comment:
A minor point, but you can put `HelpTextProps, Validation` onto the CheckboxGroupProps instead (you actually already have `Validation` on the CheckboxGroupProps.